web/client/epics/maptemplates.js

```diff
@@ -107,9 +107,10 @@ export const mergeTemplateEpic = (action$, store) => action$
                 const groups = groupsSelector(state);
                 const backgrounds = backgroundListSelector(state);
                 const textSearchConfig = textSearchConfigSelector(state);
+                const bookmarkSearchConfig = bookmarkSearchConfigSelector(state);
                 const additionalOptions = mapOptionsToSaveSelector(state);
 
-                const currentConfig = MapUtils.saveMapConfiguration(map, layers, groups, backgrounds, textSearchConfig, additionalOptions);
+                const currentConfig = MapUtils.saveMapConfiguration(map, layers, groups, backgrounds, textSearchConfig, bookmarkSearchConfig, additionalOptions);
```

Comment:
will this change need a dedicated unit test?

Reply:
Same as above. Kindly let me know if you think otherwise